coderd/apidoc/docs.go

```diff
@@ -789,6 +894,19 @@ const docTemplate = `{
                 "type": "boolean"
             }
         },
+        "codersdk.BuildInfoResponse": {
+            "type": "object",
+            "properties": {
+                "external_url": {
+                    "description": "ExternalURL is a URL referencing the current Coder version.  For production\nbuilds, this will link directly to a release.  For development builds, this\nwill link to a commit.",
```

Comment:
Mostly curious: why are there newlines in the description?

Reply:
This is the way how [swag](https://github.com/swaggo/swag) converts Go comments into `swagger.json`. I'm removing those in postprocessing, so that they won't land in our docs.